docs/src/installation.md

```diff
@@ -64,12 +69,22 @@ to NixOS via [Flakes][nix-flakes].
     stylix.url = "github:danth/stylix";
   };
 
-  outputs = { darwin, nixpkgs, stylix, ... }: {
-    darwinConfigurations."«hostname»" = darwin.lib.darwinSystem {
-      system = "aarch64-darwin";
-      modules = [ stylix.darwinModules.stylix ./configuration.nix ];
+  outputs =
+    {
+      darwin,
+      nixpkgs,
```

Comment:
Looks like this function doesn't actually need `nixpkgs`. Removing this may allow the arguments to collapse to one line again.

Reply:
Should we not keep it to simplify copy-pasting, assuming that most people access `nixpkgs` that way?

Reply:
Almost all Stylix users will have an existing configuration anyway, so I doubt many people will be copy-pasting directly from the docs. This is just showing the minimum necessary to use it.